casper/src/test/scala/coop/rchain/casper/batch1/MultiParentCasperMergeSpec.scala

```diff
@@ -229,61 +209,61 @@ class MultiParentCasperMergeSpec
-    val allMergeabilityCases = {
-      val events = List(
-        "!X",
-        "!4",
-        "!C",
-        "4X",
-        "4!",
-        //"4!!",
-        "PX",
-        "P!",
-        "P!!",
-        //"!!X",
-        //"!!4",
-        //"!!C",
-        "CX",
-        "C!"
-        //"C!!"
-      )
-
-      val pairs    = events.combinations(2)
-      val diagonal = events.map(x => List(x, x))
-      val cases    = (pairs ++ diagonal).toList
-
-      def isComm(s: String) = !s.contains("X")
-
-      def makeVolatile(s: String): List[String] = isComm(s) match {
-        case false => List(s)
-        case true  => List(s, s"($s)")
-      }
-
-      def makeVolatiles(v: List[String]): List[List[String]] =
-        for {
-          a <- makeVolatile(v(0))
-          b <- makeVolatile(v(1))
-        } yield List(a, b)
-
-      val withVolatiles = cases.flatMap(makeVolatiles)
-
-      // TODO: Do not filter out missing cases
-      withVolatiles
-        .map(_.mkString(" "))
-        .toSet
-    }
-
-    val testedMergeabilityCases =
-      (baseMergeabilityCases ++ peekMergeabilityCases)
-        .map(_._1)
-    withClue(s"""Missing cases: ${allMergeabilityCases
-      .diff(testedMergeabilityCases.toSet)
-      .toList
-      .sorted
-      .mkString(", ")}\n""") {
-      testedMergeabilityCases should contain allElementsOf allMergeabilityCases
-    }
-  }
+//  "This spec" should "cover all mergeability cases" in {
+//    val allMergeabilityCases = {
+//      val events = List(
+//        "!X",
+//        "!4",
+//        "!C",
+//        "4X",
+//        "4!",
+//        "4!!",
+//        "PX",
+//        "P!",
+//        "P!!",
+//        "!!X",
+//        "!!4",
+//        "!!C",
+//        "CX",
+//        "C!",
+//        "C!!"
+//      )
+//
+//      val pairs    = events.combinations(2)
+//      val diagonal = events.map(x => List(x, x))
+//      val cases    = (pairs ++ diagonal).toList
+//
+//      def isComm(s: String) = !s.contains("X")
+//
+//      def makeVolatile(s: String): List[String] = isComm(s) match {
+//        case false => List(s)
+//        case true  => List(s, s"($s)")
+//      }
+//
+//      def makeVolatiles(v: List[String]): List[List[String]] =
+//        for {
+//          a <- makeVolatile(v(0))
+//          b <- makeVolatile(v(1))
+//        } yield List(a, b)
+//
+//      val withVolatiles = cases.flatMap(makeVolatiles)
+//
+//      // TODO: Do not filter out missing cases
+//      withVolatiles
+//        .map(_.mkString(" "))
+//        .toSet
+//    }
+//
+//    val testedMergeabilityCases =
+//      (MergeabilityRules.baseMergeabilityCases ++ MergeabilityRules.peekMergeabilityCases)
+//        .map(_._1)
+//    withClue(s"""Missing cases: ${allMergeabilityCases
+//      .diff(testedMergeabilityCases.toSet)
+//      .toList
+//      .sorted
+//      .mkString(", ")}\n""") {
+//      testedMergeabilityCases should contain allElementsOf allMergeabilityCases
+//    }
+//  }
```

Comment:
Add reference to an issue where is tracked.

Reply:
https://github.com/rchain/rchain/issues/3277#issuecomment-798109618 but this particular test is a duplicate for mergeablilySpec I believe